tests/integration/cli/test_batch.py

```diff
@@ -18,7 +18,7 @@ def test_batch_run(self):
         '''
         Tests executing a simple batch command to help catch regressions
         '''
-        ret = 'Executing run on [\'sub_minion\']'
+        ret = 'Executing run on [u\'sub_minion\']'
```

Comment:
This fails on Python 3, since the `u` prefix is not used for unicode strings. The way I've gotten around this in other tests is to use `repr()`:

```python
ret = 'Executing run on [{0}]'.format(repr('sub_minion'))
```

This would apply to the other instances where lines have been changed below as well.

Reply:
Fixed in 71a1d2b.